bindings/java/Cargo.toml

```diff
@@ -73,7 +74,8 @@ services-all = [
   "services-mysql",
   "services-redb",
   "services-redis",
-  "services-rocksdb",
+  # FIXME has problem during cross compiling.
+  # "services-rocksdb",
```

Comment:
Any background or error report? Or we may create an issue for it.

Reply:
Local test only, let's remove this.